spec/rspec/matchers/built_in/include_spec.rb

```diff
@@ -111,6 +111,20 @@ def hash.send; :sent; end
     end
   end
 
+  shared_context "time range" do
+    let(:fmt) { RSpec::Support::ObjectFormatter }
+    let(:now) { Time.now.utc }
+    let(:start) { now - 20 }
+    let(:finish) { now - 10 }
+    let(:range) { (start..finish) }
+  end
+
+  shared_context "only runs for modern rubies" do
+    around(:example) do |example|
+      example.run if RUBY_VERSION >= "2.1.9"
```

Comment:
Can we change this for an expected error?

Reply:
@JonRowe Definitely!

I just pushed a PR that runs examples for all ruby versions so I can see the error across older rubies (the retention window for this PR's [build logs](https://github.com/rspec/rspec-expectations/runs/3797092128?check_suite_focus=true) has elapsed).

Mind kicking off another build, I'll see which errors crop up and figure out where to go from there?

Reply:
Done, I wish Github remembered approvals, I assume it goes off "merged commiters"

Reply:
@JonRowe Awesome, I just pushed those changes (it looks like old rubies raise a `TypeError` so that's what I now expect to get raised for those versions).  

Mind kicking off one more build?

Also, I'm not 100% sure that my naming for the shared examples is best.  Open to suggestion there!